### DISCUSSION TITLE
need help getting http to be https!

### DISCUSSION THREAD

**Discussion by Mrs-Feathers** [netbox-community/netbox-docker#1104] [Getting Help]
Hello! my set up is a server that runs netbox and then we have a reverse proxy with authentik on it. this supplies the OIDC we use for the login. our nginx reverse proxy supplies https and certs and stuff so i dont want to add https support or anything, i just want to set the base_url or something to be with https, otherwise it passes http to authentik and the login errors out. how can i trick netbox into thinking its already https? most apps just have a base_url setting but i can't find it for netbox.. only base_path which isn't useful for this issue.

**Comment by alexandrud** [comment]
You might try this:
```
SOCIAL_AUTH_REDIRECT_IS_HTTPS = False
On projects behind a reverse proxy that uses HTTPS, the redirect URIs can have the wrong schema (http:// instead of https://) if the request lacks the appropriate headers, which might cause errors during the auth process. To force HTTPS in the final URIs set this setting to True
```
The full list is here: https://python-social-auth.readthedocs.io/en/latest/configuration/settings.html#processing-requests-and-redirects
